Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -1,5 +1,71 @@
 ## iOS Platform APIs
 
+### Testing App Permissions
+
+#### Overview
+iOS makes all mobile applications run under the `mobile` user. Each application is sandboxed and limited using policies enforced by the Trusted BSD mandatory access control framework. These policies are called profiles and all third-party applications use on generic sandbox profile: the container permission list. <TODO: ADD REFERENCE TO
+http://www.icri-sc.org/fileadmin/user_upload/Group_TRUST/PubsPDF/sandscout-final-ccs-2016.pdf >.
+
+On iOS, apps need to request permission to the user for accessing one of the following data or resources: Bluetooth peripherals, Calendar data, Camera, Contacts, Health sharing, Health updating, HomeKit, Location, Microphone, Motion, Music and the media library, Photos, Reminders, Siri, Speech recognition, and the TV provider.<TODO: add reference to https://developer.apple.com/library/archive/documentation/iPhone/Conceptual/iPhoneOSProgrammingGuide/ExpectedAppBehaviors/ExpectedAppBehaviors.html#//apple_ref/doc/uid/TP40007072-CH3-SW7 >
```

Comment:
Please list the ressources, so it's easier to read

Reply:
👍